### ISSUE TITLE
Error when changing hidden_size

### ISSUE THREAD

**Issue by robinjia** [jiasenlu/HieCoAttenVQA#36]
I was able to get training to work with default hyperparameters on VQA, but the accuracy was lower than reported in the paper (I got about 50%).  I saw that in the paper, a hidden size of 1024 was used for VQA, instead of the default of 512.  When I set -hidden_size 1024 when running `train.lua`, I got the following error:

/user/torch/install/bin/luajit: /user/torch/install/share/lua/5.1/nn/CAddTable.lua:16: bad argument #2 to 'add' (sizes do not match at /user/torch/extra/cutorch/lib/THC/generated/../generic/THCTensorMathPointwise.cu:217)
stack traceback:
	[C]: in function 'add'
	/user/torch/install/share/lua/5.1/nn/CAddTable.lua:16: in function 'func'
	/user/torch/install/share/lua/5.1/nngraph/gmodule.lua:345: in function 'neteval'
	/user/torch/install/share/lua/5.1/nngraph/gmodule.lua:380: in function 'forward'
	./misc/ques_level.lua:119: in function 'forward'
	train.lua:258: in function 'lossFun'
	train.lua:311: in main chunk
	[C]: in function 'dofile'
	/user/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00406670
